pynest/nest/hl_api.py

```diff
@@ -1058,11 +1059,14 @@ def Connect(pre, post, conn_spec=None, syn_spec=None, model=None):
     of NEST's built-in synapse models or a user-defined model created via CopyModel().
     If 'model' is not specified the default model 'static_synapse' will be used.
 
-    All other parameters can be scalars or distributions. In the case of scalar 
-    parameters, all keys must be doubles except for 'receptor_type' which must be
-    initialised with an integer. Any distributed parameter must be initialised with 
-    a further dictionary specifying the distribution type ('distribution', e.g. 'normal') 
-    and any distribution-specific parameters (e.g. 'mu' and 'sigma').
+    All other parameters can be scalars, arrays or distributions. 
+    In the case of scalar parameters, all keys must be doubles except for 'receptor_type' which must be
+    initialised with an integer. 
+    Parameter arrays are only avaibale for the rules 'one_to_one' and 'all_to_all'. For 'one_to_one' the
+    array has to be a one-dimensional NumPy array with length len(pre) For 'all_to_all' the array has
+    to be a two-dimensional NumPy array with shape len(post) x len(pre).
```

Comment:
For extra clarity, I would write "with shape `(len(post), len(pre))`.", i.e., give a tuple as returned by `.shape`. The documentation should also state explicitly that rows represent target, columns source neurons.